ui/src/institutions/components/InstitutionAddress.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Map, List } from 'immutable';
+import InlineList, {
+  InlineUL,
+  SEPARATOR_COMMA,
+} from '../../common/components/InlineList';
+
+function postalAddressContainsString(postalAddress, string) {
```

Comment:
it's better not use keywords like `string` and `type`, incase we'd like to add some type checking with `TypeScript`.